bundles/org.openhab.binding.miio/src/main/resources/database/deerma.humidifier.mjjsq.json

```diff
@@ -0,0 +1,93 @@
+{
+	"deviceMapping": {
+		"id": [
+			"deerma.humidifier.mjjsq"
+		],
+		"channels": [
+			{
+				"property": "OnOff_State",
+				"friendlyName": "Power",
+				"channel": "power",
+				"type": "Switch",
+				"refresh": true,
+				"actions": [
+					{
+						"command": "Set_OnOff",
+						"parameterType": "ONOFF"
+					}
+				]
+			},
+			{
+				"property": "Humidifier_Gear",
+				"friendlyName": "Mode",
+				"channel": "mode",
+				"type": "String",
+				"refresh": true,
+				"actions": [
+					{
+						"command": "Set_HumidifierGears",
+						"parameterType": "STRING"
+					}
+				]
+			},
+			{
+				"property": "Humidity_Value",
+				"friendlyName": "Humidity",
+				"channel": "humidity",
+				"type": "Number",
+				"refresh": true,
+				"actions": [
+					{
+						"command": "Set_HumiValue",
+						"parameterType": "NUMBER"
+					}
+				]
+			},
+			{
+				"property": "HumiSet_Value",
+				"friendlyName": "Humidity Setting",
+				"channel": "humidity_set",
+				"type": "Number",
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "Led_State",
+				"friendlyName": "LED",
+				"channel": "ked",
```

Comment:
Could it be an error in channel name, i.e. should be led instead of ked

Reply:
yep